resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -167,12 +167,12 @@ const CharacterTable = ({
                                 page * rowsPerPage,
                                 page * rowsPerPage + rowsPerPage,
                             )
-                            .map((row: RowData, index: number) => {
-                                const isItemSelected = isSelected(row.cname)
+                            .map((row: CharacterRowData, index: number) => {
+                                const isItemSelected = isSelected(row.name)
                                 const labelId = `enhanced-table-checkbox-${index}`
                                 return (
                                     <TableRow
-                                        key={row.cname}
+                                        key={row.name}
```

Comment:
i believe key should be the id from backend somehow, the name is not an unique identifier